src/Common/tests/System/Net/Capability.Sockets.cs

```diff
@@ -47,5 +47,10 @@ public static bool IPv4Support()
         {
             return Socket.OSSupportsIPv4;
         }
+
+        public static bool UnixDomainSocketsSupport()
+        {
+            return !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
I can that to SocketProtocolSupportPal.Unix and  SocketProtocolSupportPal.Windows the same way OSSupportsIPv4 works.